exec/rpc/src/main/java/org/apache/drill/exec/rpc/RequestIdMap.java

```diff
@@ -84,7 +115,7 @@ public void operationComplete(ChannelFuture future) throws Exception {
       if (!future.isSuccess()) {
         removeFromMap(coordinationId);
         if (future.channel().isActive()) {
-           throw new RpcException("Future failed") ;
+          throw new RpcException("Future failed");
```

Comment:
Since the future did not succeed, should this `setException(future.cause())`? There would be no outcome for the `handler` otherwise, right?